### ISSUE TITLE
Autocomplete permissions and roles

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/webform_civicrm#40]
I was looking into the autocomplete functionality in relation to permissions, and although this isn't the issue I was looking for, I think the roles logic needs some updating for Backdrop. See if this looks right to you (PR coming).

---
PR by @laryn: https://github.com/backdrop-contrib/webform_civicrm/pull/41

**Comment by laryn** [comment]
I take it back. This does seem to solve the problem I was having. 👍 

Edit: For clarity, the issue I was having is that after choosing some filters to limit the contacts that show up (in the settings for the existing contact autocomplete field), I unchecked the "Enforce permissions" box. But the anonymous user filling out the form still couldn't autocomplete any of the valid options. This PR seems to fix that.

**Comment by jackaponte** [comment]
Thanks for this, @laryn! I hope to review this PR in the next couple of days.

**Comment by jackaponte** [comment]
Tested this PR thusly:

1. In CiviCRM settings for a webform, added a new Existing Contact field
2. In Webform settings for the new Existing Contact field, set "Form Widget" to Autocomplete, set a "Type of Individual" filter, unchecked "Enforce Permissions"
3. Visited the form as an anonymous user was able to autocomplete one of the matching valid options, whereas without this PR checked out no options were available to the anonymous user.

I'd say this PR is ready to be merged in, but just want to double-check that I've checked the right thing here, @laryn!

**Comment by laryn** [comment]
Yes, that sounds like a similar set up as I was working from, although rather than a "Type of Individual" filter, I was using a CiviCRM group as a filter. Thanks for testing/verifying!

**Comment by jackaponte** [comment]
Thanks for confirming, @laryn! I've merged in your pull request.

@herbdool, think it's OK for me to create a bug fix release (1.x-4.28.1) that includes this commit? Or should we hold off on a new release for any reason?

**Comment by laryn** [comment]
Thanks!

**Comment by jackaponte** [comment]
> @herbdool, think it's OK for me to create a bug fix release (1.x-4.28.1) that includes this commit? Or should we hold off on a new release for any reason?

I don't want this question to get lost since the issue is now closed; any thoughts on whether I should create a bug fix release soon to fix this one bug, or whether we should wait on a new release? (Reasons I can see for waiting: to have more bug fixes to commit before issuing a new release; to set up a schedule for bug fix releases)

Should I separate this question out into a new meta-issue about how we want to manage module maintenance? :)

**Comment by herbdool** [comment]
I think it's ok to create a release

**Comment by jackaponte** [comment]
> I think it's ok to create a release

Cool--I'm going to get to work on that in the coming couple of weeks. I'll open a separate issue for that if need be!
